genie-docs/src/docs/asciidoc/_properties.adoc

```diff
@@ -35,9 +35,14 @@ Whereas static properties values are bound during application startup and do not
 |
 |yes
 
-|genie.agent.launcher.local.executable
-|The system command that the launcher should use to launch an agent process. Ordered list of arguments.
-|java -jar /tmp/genie-agent.jar
+|genie.agent.launcher.local.agent-jar-path
```

Comment:
These changes seem separate from the commit message/purpose.

Additionally what do you feel is the value of breaking it up like this and also putting in all this string replacement convention stuff? you don't really document what the available replacement options are. I understand this is an attempt to break it out of hardcoding (which is definitely good) but we may want to abstract it better?

Internally the executable isn't even a jar so it's a little strange to rely so much on `agent-jar-path`

Reply:
Now all in one commit. And the description for this property is more clear on its purpose.